defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+#openstack_undercloud_version: 13
+openstack_undercloud_become: true
+openstack_undercloud_become_user: root
+
+openstack_undercloud_ssl_key_size: 2048
+openstack_undercloud_ssl_key_type: RSA
+openstack_undercloud_local_ip: |-
```

Comment:
I'd rename this var to `openstack_undercloud_local_ip4` to avoid confusion with anything related to that other protocol.

Reply:
"local_ip" is the name of the value in undercloud.conf that this value is inserted to. I kept this and several other values in here matching.

Reply:
I like the consistency!